src/main/java/com/twilio/authy2fa/servlet/authentication/LogInServlet.java

```diff
@@ -17,23 +19,28 @@
 @WebServlet(urlPatterns = {"/login"})
 public class LogInServlet extends HttpServlet {
 
+    private static final Logger  LOGGER = LoggerFactory.getLogger(LogInServlet.class);
+
     private final SessionManager sessionManager;
     private final UserService userService;
-    private final Client client;
+    private final AuthyApiClient client;
+    private final ApprovalRequestService approvalRequestService;
 
     @SuppressWarnings("unused")
     public LogInServlet() {
         this(
                 new SessionManager(),
                 new UserService(),
-                new Client(System.getenv("AUTHY_API_KEY"))
-        );
+                new AuthyApiClient(System.getenv("AUTHY_API_KEY")),
```

Comment:
@hortega :: Instead of getting the `AUTHY_API_KEY` value from the environment, I think it would be better to reuse the available method `ConfigurationService.authyApiKey()` here.

```java
new AuthyApiClient((new ConfigurationService()).authyApiKey()),
```

wdyt?

Reply:
Definitely! Good catch @well1791